### PR TITLE
Email Digest Fronted Work

### LINKED ISSUE #2838: Simple Email Digest Opt-In Work
## Description

A simple UI on the Notification Settings Page to set the Users Email Digest Frequency. It should set the frequency of the email digest as just (Never, or Weekly) for the moment. User model attribute: `emailNotificationInterval` enum needs to be updated from (Daily, Never) to (Weekly, Never). Migration needed.

WIP Design:
https://www.figma.com/file/GAITgpjiE4xamopqhnLnQj/Email-Activity-Updates?node-id=99%3A10534&t=6DEfACVTgQ7pGM6g-0

### PR DESCRIPTION
**Pull request by alexyoung23j**
## Description

Email Digest settings page. Lets user select between "never" and "weekly". If no email is registered to the user, provides an input section for that email to be added. 

Closes #2838 

## Does this PR affect any server routes?
chose YES or NO

## Have you added the issue number here?
(In the right sidebar, under "development")

### CODE REVIEW

File: packages/commonwealth/server/routes/writeUserSetting.ts [hunk 1]
```diff
@@ -2,7 +2,13 @@ import { AppError } from 'common-common/src/errors';
 import type { NextFunction, Request, Response } from 'express';
 import type { DB } from '../models';
 
-const VALID_DIGEST_INTERVALS = ['daily', 'never'];
+const VALID_DIGEST_INTERVALS = [
+  'never',
+  'weekly',
+  'daily',
+  'biweekly',
```
Comment (zakhap) [inline packages/commonwealth/server/routes/writeUserSetting.ts:9]: not sold on biweekly for every 2 weeks. @jnaviask for the best solution here

**Review by zakhap — DISMISSED** [review]
UI Seems to work just fine 💪 question about the naming of the intervals

File: packages/commonwealth/server/routes/writeUserSetting.ts (hunk 1 above)
Comment (jnaviask) [reply packages/commonwealth/server/routes/writeUserSetting.ts:9]: 2weeks?

**Review by CowMuon — APPROVED** [review]
LGTM

File: packages/commonwealth/server/routes/writeUserSetting.ts (hunk 1 above)
Comment (CowMuon) [reply packages/commonwealth/server/routes/writeUserSetting.ts]: We need to be clear on what biweekly means here. 

TBC: biweekly can mean either "twice a week" or "every 2 weeks" (don't blame me I didn't do that) but is more commonly used for latter. 

However I kind of have a niggle here in that the option ordering makes no sense: either biweekly or daily should go between never/weekly, but the above order here doesn't reflect a sequence that makes sense for either definition.

**Comment by zakhap** [comment]
@CowMuon the fix was fixed

**Pull request merged**